modules/nodes-nodes-rebooting-gracefully.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-rebooting.adoc
+
+[id="nodes-nodes-rebooting-gracefully_{context}"]
+= Rebooting a node gracefully
+
+Before rebooting a node, it is recommended to take a backup of etcd data to avoid any data loss on the node.
+
+.Procedure
+
+To perform a graceful restart of a node:
+
+. Mark the node as unschedulable:
++
+[source,terminal]
+----
+$ oc adm cordon <node1>
+----
++
+. Drain the node to remove all the running pods:
++
+[source,terminal]
+----
+$ oc adm drain <node1>
+----
++
+. Access the node in debug mode:
++
+[source,terminal]
+----
+$ oc debug node/<node1>
+----
++
+. Restart the node:
++
+[source,terminal]
+----
+$ systemctl reboot
+----
++
+. Mark the node as schedulable after the reboot is complete:
++
+[source,terminal]
+----
+$ oc adm uncordon <node1>
+----
++
+. Verify that the node is ready:
++
+[source,terminal]
+----
+$ oc get node <node1>
+----
++
+.Example output
+[source,terminal]
+----
+NAME    STATUS  ROLES    AGE     VERSION
+<node1> Ready   worker   6d22h   v1.18.3+b0068a8
+----
+
+
+
+.Additional resources
```

Comment:
If you do not have any additional resources to add to this section, please remove the heading.

Reply:
Addressed.